tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -11,14 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from cStringIO import StringIO
-import json
 
+import datetime
+import json
 
-def format_struct(struct):
```

Comment:
This is a duplicate of the same function in another file, but this one wasn't used. removed